WPMedia/fusion-news-theme-blocks
# updated max-height and max-width of author image to 88px

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

Author-bio image height and width to be 88px(spacing-13).

## Jira Ticket

- [TMEDIA-717](https://arcpublishing.atlassian.net/browse/TMEDIA-717)

## Acceptance Criteria

Author-bio image height and width to be 88px(spacing-13).
https://arcpublishing.atlassian.net/browse/TMEDIA-717?focusedCommentId=803666

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-717-image-height-width`
2. Click on short-author bio block and inspect the. image to check the height and with

## Effect Of Changes

### Before
<img width="1675" alt="Screen Shot 2022-09-07 at 4 59 49 PM" src="https://user-images.githubusercontent.com/83021791/188978373-632020e4-b4c8-4663-b3d3-eb08aaa72f08.png">


### After
<img width="1694" alt="Screen Shot 2022-09-07 at 5 00 08 PM" src="https://user-images.githubusercontent.com/83021791/188978456-9df18ece-bdb6-407e-a54a-4b6d402658fa.png">


## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
